src/app/how-council-works/page.tsx

```diff
@@ -0,0 +1,257 @@
+'use client';
+
+import { TooltipProvider } from '@radix-ui/react-tooltip';
```

Comment:
If we re-export this in the tooltip component we can import them beside one another